consensus/safety-rules/src/tests/networking.rs

```diff
@@ -8,7 +8,9 @@ use libra_types::validator_signer::ValidatorSigner;
 fn test_reconnect() {
     let signer = ValidatorSigner::from_int(0);
     let storage = test_utils::test_storage(&signer);
-    let safety_rules_manager = SafetyRulesManager::new_thread(storage, false);
+    // test value for network timeout, in milliseconds.
+    let network_timeout = 5_000;
+    let safety_rules_manager = SafetyRulesManager::new_thread(storage, false, network_timeout);
```

Comment:
Nit: If this is testing the timeout 1 second might be better to keep the tests rolling.

Reply:
Legacy test timeout value was 5 seconds.  I'll leave tuning to follow ups.